# Request a plan to the given path

geometry_msgs/Pose[] waypoints

---
bool success
